OpenRA.Game/Scripting/ScriptContext.cs

```diff
@@ -158,7 +159,7 @@ public sealed class ScriptContext : IDisposable
 				.ToArray();
 
 			runtime.Globals["GameDir"] = Platform.GameDir;
-			runtime.DoBuffer(Game.ModData.ModFiles.Open(Platform.ResolvePath(".", "lua", "scriptwrapper.lua")).ReadAllText(), "scriptwrapper.lua").Dispose();
+			runtime.DoBuffer(File.Open(Platform.ResolvePath(".", "lua", "scriptwrapper.lua"), FileMode.Open).ReadAllText(), "scriptwrapper.lua").Dispose();
```

Comment:
This is a core engine file (like the glsl shaders), so isn't referenced from the mod yaml.